Makefile

```diff
@@ -0,0 +1,43 @@
+TEST=test
+PDB_PATH=$(shell find /usr/lib{,64} -type f -executable -name pdb.py 2> /dev/null)
+DATA=$(shell echo $(MODULE) | sed 's/\(foreman\|katello\)_//; s/_/-/g')
+MODULE_PATH=modules/$(MODULE).py
+DEBUG_DATA_PATH=test/debug_data/$(DATA).json
+DEBUG_OPTIONS=-m $(MODULE_PATH) -a @$(DEBUG_DATA_PATH) -D $(PDB_PATH)
+
+default: help
+help:
+	@echo "Please use \`make <target>' where <target> is one of:"
+	@echo "  help         to show this message"
+	@echo "  lint         to run flake8 and pylint"
+	@echo "  test         to run unit tests"
+	@echo "  debug        debug a module using the ansible hacking module"
+	@echo "  setup        to set up test, lint, and debugging"
+	@echo "  test-setup   to install test dependencies"
+	@echo "  debug-setup  to set up the ansible hacking module"
+
+lint:
+	pycodestyle --ignore=E402,E722 --max-line-length=160 modules test
```

Comment:
Should this be `$(TEST)`?

Reply:
Nah, that variable is so that you can do `make test TEST=test/test_product.py`. I guess we could add a `$(LINT)` variable in case a user wants to override what gets checked in linting, but I don't see it as very useful, mostly because linting everything takes less than a second:

```
$ time make lint
pycodestyle --ignore=E402,E722 --max-line-length=160 modules test

real    0m0.391s
user    0m0.376s
sys     0m0.015s
```

Compare that with the tests (only 2 right now):

```
$ time make test
pytest test
=============================================================================================== test session starts ===============================================================================================
platform linux2 -- Python 2.7.5, pytest-3.2.3, py-1.4.34, pluggy-0.4.0
rootdir: /home/vagrant/foreman-ansible-modules, inifile:
collected 2 items                                                                                                                                                                                                  

test/test_organization.py .
test/test_product.py .

============================================================================================ 2 passed in 43.16 seconds ============================================================================================

real    0m43.381s
user    0m9.205s
sys     0m0.948s
```

Reply:
Good point. I was unsure what the variable would be for and came up with overriding the directory containing the tests by say 'new-tests'.
On the other hand, it might be convenient to specify the test in question without the directory.